oar/README.md

```diff
@@ -106,3 +107,10 @@ $ oar -r $release-version drop-bugs
 ```
 $ oar -r $release-version change-advisory-status
 ```
+12. This command checks for blocking security alerts across all RHSA advisories for the release. It automatically updates the worksheet dropdown (cell B29) and sends notifications to the QE release team if blocking alerts are detected. The command supports different output formats and notification options
+```
+$ oar -r $release-version check-blocking-sec-alerts
+$ oar -r $release-version check-blocking-sec-alerts --check-only
+$ oar -r $release-version check-blocking-sec-alerts --output json
```

Comment:
is there any other program will consume the output of this cmd, I think this option is useless

Reply:
Not in our tool but in case if we have to give detailed output to prod-sec team (If in special case) this could be useful for debugging I think